config.json

```diff
@@ -242,6 +242,19 @@
         "lists",
         "math"
       ]
+    },
+    {
+      "slug": "pangram",
+      "uuid": "0ea556b0-afcd-479d-bbad-68bc4556c1d0",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
I think having this unlocked after anagram would fit well, what do you think?

Reply:
Yeah, that seems like a good idea to me. I'll make the update!